tensorcircuit/noisemodel.py

```diff
@@ -57,40 +59,79 @@ def add_noise(
         :param qubit: the list of noisy qubit, defaults to None, indicating applying the noise channel on all qubits
         :type qubit: Optional[Sequence[Any]], optional
         """
-        if gate_name != "readout":
-            gate_name = AbstractCircuit.standardize_gate(gate_name)
+        if gate_name == "readout":
+            # probably need to refactor readout error
+            assert qubit is None
```

Comment:
we should support different readout on different qubit, this is a level-1 requirements

Reply:
should be auto satisfied